<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardDialog</class>
 <widget class="QDialog" name="WizardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>939</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="17" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <widget class="QPushButton" name="pbLoadData">
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbReset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbMultiplexing">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Multiplexing</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="color_layout">
       <item>
        <widget class="QPushButton" name="pbSingleColor">
         <property name="text">
          <string>Single fluorophore</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbColorUnmixer">
         <property name="text">
          <string>Unmixer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="fluorophores_layout">
       <item>
        <widget class="QLabel" name="lbActiveColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Active fluorophore</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmActiveColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>All</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbFilters">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Filters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbTimeInspector">
       <property name="text">
        <string>Time Inspector</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbAnalyzer">
       <property name="text">
        <string>Analyzer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbEFOFiltering">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>EFO filtering</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="efo_lower_bound_layout">
       <item>
        <widget class="QLabel" name="lbEFOLowerBound">
         <property name="text">
          <string>Lower bound (Hz)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leEFOLowerBound">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="efo_upper_bound_layout">
       <item>
        <widget class="QLabel" name="lbEFOUpperBound">
         <property name="text">
          <string>Upper bound (Hz)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leEFOUpperBound">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pbEFOFilter">
       <property name="text">
        <string>FIlter EFO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbCFRFiltering">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CFR filtering</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="cfr_lower_bound_layout">
       <item>
        <widget class="QLabel" name="lbCFRLowerBound">
         <property name="text">
          <string>Lower bound</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leCFRLowerBound">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="cfr_upper_bound_layout">
       <item>
        <widget class="QLabel" name="lbCFRUpperBound">
         <property name="text">
          <string>Upper bound</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leCFRUpperBound">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pbCFRFilter">
       <property name="text">
        <string>FIlter CFR</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbSaveData">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbExportData">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pbLoadData</tabstop>
  <tabstop>pbReset</tabstop>
  <tabstop>pbSingleColor</tabstop>
  <tabstop>pbColorUnmixer</tabstop>
  <tabstop>cmActiveColor</tabstop>
  <tabstop>pbTimeInspector</tabstop>
  <tabstop>pbAnalyzer</tabstop>
  <tabstop>leEFOLowerBound</tabstop>
  <tabstop>leEFOUpperBound</tabstop>
  <tabstop>pbEFOFilter</tabstop>
  <tabstop>leCFRLowerBound</tabstop>
  <tabstop>leCFRUpperBound</tabstop>
  <tabstop>pbCFRFilter</tabstop>
  <tabstop>pbSaveData</tabstop>
  <tabstop>pbExportData</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
